--- origsrc/gnome-applets-3.18.1/windowpicker/src/Makefile.am	2015-09-30 12:40:25.000000000 -0500
+++ src/gnome-applets-3.18.1/windowpicker/src/Makefile.am	2015-11-15 21:58:16.633027100 -0600
@@ -18,7 +18,7 @@ libwindow_picker_applet_la_CFLAGS = \
 	$(NULL)
 
 libwindow_picker_applet_la_LDFLAGS = \
-	-module -avoid-version \
+	-module -avoid-version -no-undefined \
 	$(WARN_LDFLAGS) \
 	$(AM_LDFLAGS) \
 	$(NULL)
