--- origsrc/gnome-applets-3.5.92/null_applet/null_applet.c	2012-09-19 16:46:37.000000000 -0500
+++ src/gnome-applets-3.5.92/null_applet/null_applet.c	2013-08-07 01:37:31.512379500 -0500
@@ -29,6 +29,7 @@
 #include <gconf/gconf-client.h>
 #include "null_applet.h"
 
+GType null_applet_get_type (void);
 G_DEFINE_TYPE(NullApplet, null_applet, PANEL_TYPE_APPLET)
 
 static void
--- origsrc/gnome-applets-3.5.92/trashapplet/src/trashapplet.c	2012-09-19 16:46:37.000000000 -0500
+++ src/gnome-applets-3.5.92/trashapplet/src/trashapplet.c	2013-08-07 01:36:18.199186200 -0500
@@ -51,6 +51,7 @@ typedef struct
   gint items;
 } TrashApplet;
 
+GType trash_applet_get_type (void);
 G_DEFINE_TYPE (TrashApplet, trash_applet, PANEL_TYPE_APPLET);
 #define TRASH_TYPE_APPLET (trash_applet_get_type ())
 #define TRASH_APPLET(obj) (G_TYPE_CHECK_INSTANCE_CAST ((obj), \
