gweather applet depends on GWeatherXML API, removed in 3.7/3.8

--- origsrc/gnome-applets-3.5.92/configure.in	2012-09-19 20:07:34.000000000 -0500
+++ src/gnome-applets-3.5.92/configure.in	2013-08-07 01:48:01.662422000 -0500
@@ -251,7 +251,7 @@ AC_SUBST(XML2_LIBS)
 
  dnl -- check for libgweather (required for gweather applet) ------------------
 build_libgweather_applets=false
-PKG_CHECK_MODULES(LIBGWEATHER, gweather-3.0 >= $GWEATHER_REQUIRED,
+PKG_CHECK_MODULES(LIBGWEATHER, gweather-3.0 >= $GWEATHER_REQUIRED gweather-3.0 < 3.7,
 	          build_libgweather_applets=true,
 		  AC_MSG_WARN([libgweather not found. Not building the weather applet.]))
 AC_SUBST(LIBGWEATHER_CFLAGS)
